sql/core/src/test/scala/org/apache/spark/sql/CountMinSketchAggQuerySuite.scala

```diff
@@ -61,4 +61,23 @@ class CountMinSketchAggQuerySuite extends QueryTest with SharedSparkSession {
 
     assert(sketch == reference)
   }
+
+  test("count-min sketch with named-arguments") {
```

Comment:
nit: we can remove this end-to-end test if `named-function-arguments.sql` includes `count_min_sketch`